DEPS

```diff
@@ -11,17 +11,25 @@ Core dependencies
 
 Essential:
 
-* python (== 2.7.x)
+* python3 (>= 3.6)
 * bsddb (distributed with stdlib) or bsddb3
 * gtk+ >= 3.10
 * gstreamer >= 1.4
 * gstreamer-plugins-base >= 1.6
 * gstreamer-plugins-good >= 1.4
-* mutagen (>= 1.10)
-* dbus-python
-* pygobject (>= 3.13.2)
```

Comment:
We probably still need to mention pygobject and its version.

Reply:
@sjohannes : Any objections?

Reply:
Hm, in that case maybe we should attach the version number to python3-gi (Debian) and python3-gobject (Fedora)? It's not a big deal, though, and I'm not sure that number is still accurate anyway.

Reply:
In practice, nobody ever tests exaile on old python3-gi versions. So I'd just fill in the oldest version our CI is testing against, that's ubuntu18.04 with python3-gi 3.22.0.

Reply:
I also updated some other dependencies accordingly (gtk3, gstreamer, …)